samples/mpsl/timeslot/src/main.c

```diff
@@ -65,6 +65,18 @@ static void error(void)
 	}
 }
 
+ISR_DIRECT_DECLARE(swi2_isr)
+{
+	/* Put callback info in the message queue. */
+	int err = k_msgq_put(&callback_msgq, &signal_callback_return_param.callback_action,
+						K_NO_WAIT);
+
+	if (err) {
+		error();
```

Comment:
Unrelated to the changes in this PR. I would suggest replacing `error()` with `k_oops()` to ensure we are not using kernel APIs when we can't.

Reply:
I may misunderstand. But k_oops() is also a kernel function, are you suggesting to define an in-house error function?